### ISSUE TITLE
Improperly escaped strings on the command line in Calc end up displaying badly

### ISSUE THREAD

**Issue by rlwhitcomb** [rlwhitcomb/utilities#171]
Consider (when :quotestrings off):
```
> c :echo `$$0` -- c:\test\files\input.txt
c:      estilesnput.txt
```

**Comment by rlwhitcomb** [comment]
This is all because of the change in behavior for the Settings stuff using "quoteControl" when :quotestrings is off.
But I really don't understand it:
Even without "quoteControl" in the "toStringValue" path in CalcUtil, I'm getting this:
```
:echo `File: ${replace($0, '\\\\', '/', all)}`
```
```
Error: Unexpected internal error near index 1
\ at line 1.
File:
```

**Comment by rlwhitcomb** [comment]
This has been fixed by recent changes for #334 .
